finals-2024-01-15/code/5094-viktor-lozanovski/src/app/movie-list/movie-list.component.ts

```diff
@@ -1,10 +1,34 @@
-import { Component } from '@angular/core';
+// movie-list.component.ts
+import { Component, OnInit } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
 
 @Component({
   selector: 'app-movie-list',
   templateUrl: './movie-list.component.html',
-  styleUrl: './movie-list.component.css'
+  styleUrls: ['./movie-list.component.css']
 })
-export class MovieListComponent {
+export class MovieListComponent implements OnInit {
```

Comment:
+3 for the component code. Fetching directly in the component should be avoided because it hard-codes a lot of things that can and will change.